policyengine_uk/parameters/gov/dwp/childcare_subsidies/free_childcare/income_thresholds.yaml

```diff
@@ -0,0 +1,23 @@
+description: Income thresholds for tax-free childcare eligibility
```

Comment:
```suggestion
description: The Department of Work and Pensions provides the tax free childcare program to household with income below this threshold, based on the age of the dependent.
```

Reply:
All of the descriptions should be full sentences The Department of Work and Pensions does xyz under the tax free childcare program